crates/graphql/src/schema/objects/nft.rs

```diff
@@ -206,6 +206,16 @@ impl TryFrom<models::NftActivity> for NftActivity {
     }
 }
 
+#[graphql_object(Context = AppContext)]
```

Comment:
Need to drop `GraphQLObject` on the NftActivity struct and add getter methods for all the fields on the struct.